v63005/inref/gtm5059.m

```diff
@@ -0,0 +1,23 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
Since this M program is the same as v63004/inref/gtm1042.m, can we use that instead of creating a duplicate M program. You can do a "cp $gtm_tst/v63004/inref/gtm1042.m gtm5059.m" in the gtm5059.csh script.